databook/_toc.yml

```diff
@@ -16,74 +16,96 @@ chapters:
-    - file: visual-behavior/VBO-ExperimentData
+    - file: physiology/visual-behavior/VBO-ExperimentData
       title: Ophys Experiment Data
-    - file: visual-behavior/VBO-Tutorial-Compare_trial_types
+    - file: physiology/visual-behavior/VBO-Tutorial-Compare_trial_types
       title: Tutorial Comparing Trial Types
-    - file: visual-behavior/VBO-Tutorial-Downloading_data_and_exploring_the_manifest
+    - file: physiology/visual-behavior/VBO-Tutorial-Downloading_data_and_exploring_the_manifest
       title: Tutorial Downloading Data
-  - file: visual-behavior/VB-Neuropixels
+  # New file here
+  - file: physiology/mFISH/mFISH                                     # No idea what this is, but here we are. May get restructured again
+    sections:
+      - file: physiology/mFISH/mFISH-Dataset                         # Also a placeholder that should get (NEEDS) actual content
+  # Which should not be neglected
+  - file: physiology/visual-coding-np/vcnp
+    sections:
+    - file: physiology/visual-coding-np/vcnp-quickstart
+    - file: physiology/visual-coding-np/vcnp-data-access
+    - file: physiology/visual-coding-np/vcnp-units
+    - file: physiology/visual-coding-np/vcnp-receptive-fields
+    - file: physiology/visual-coding-np/vcnp-lfp
+    - file: physiology/visual-coding-np/vcnp-optotagging
+    - file: physiology/visual-coding-np/vcnp-session
+    - file: physiology/visual-coding-np/vcnp-cheatsheet
+    - file: physiology/visual-coding-np/vcnp-links
+  - file: physiology/visual-behavior/VB-Neuropixels
     title: Visual Behavior Neuropixels
     sections:
-    - file: visual-behavior/VBN-Dataset
-    - file: visual-behavior/VBN-SessionData
-    - file: visual-behavior/VBN-Tutorial-Aligning_Neural_Data_to_Stimuli.ipynb
+    - file: physiology/visual-behavior/VBN-Dataset
+    - file: physiology/visual-behavior/VBN-SessionData
+    - file: physiology/visual-behavior/VBN-Tutorial-Aligning_Neural_Data_to_Stimuli.ipynb
       title: Tutorial Aligning Neural Data to Stimuli
-    - file: visual-behavior/VBN-Tutorial-Aligning_Behavior_Data_to_Task_Events.ipynb
+    - file: physiology/visual-behavior/VBN-Tutorial-Aligning_Behavior_Data_to_Task_Events.ipynb
       title: Tutorial Aligning Behavioral Data to Task Events
-    - file: visual-behavior/VBN-Tutorial-Analyzing_LFP_Data.ipynb
+    - file: physiology/visual-behavior/VBN-Tutorial-Analyzing_LFP_Data.ipynb
       title: Tutorial Analyzing LFP data
-- file: microns-em/em-background
+  - file: physiology/cell-type-lookup-table/ctlut-background
+    sections:
+    - file: physiology/cell-type-lookup-table/ctlut-session-data
+    - file: physiology/cell-type-lookup-table/ctlut-optotagging
+    - file: physiology/cell-type-lookup-table/ctlut-accessing-data
+    - file: physiology/cell-type-lookup-table/ctlut-identifying-tagged-units
+  # New files here
+  - file: physiology/np-ultra/np-ultra                     # Also a placeholder that needs actual content
```

Comment:
physiology/ephys/np-ultra
then physiology/ephys/np-ultra/np-ultra-background

Reply:
Adjusted file structure and updated ToC